src/chat/chatter.py

```diff
@@ -273,43 +274,52 @@ def updatePlayer(self):
         # Rating icon choice  (chr(0xB1) = +-)
         formatting = ("Global Rating: {} ({} Games) [{}\xb1{}]\n"
                       "Ladder Rating: {} [{}\xb1{}]")
-        self.rankItem.setToolTip(formatting.format(
+        tooltip_str = formatting.format(
             str(int(rating)),
             str(player.number_of_games),
             str(int(player.rating_mean)),
             str(int(player.rating_deviation)),
             str(int(ladder_rating)),
             str(int(player.ladder_rating_mean)),
-            str(int(player.ladder_rating_deviation))))
+            str(int(player.ladder_rating_deviation)))
 
         league = player.league
         if league is not None:
-            formatting = ("Division : {}\n"
-                          "Global Rating: {}")
-            self.rankItem.setToolTip(formatting.format(
-                league["division"],
-                str(int(rating))))
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % league["league"]))
+            icon_str, tooltip_str = league["league"], "Division : " + league["division"] + "\n" + tooltip_str
         else:
-            self.rankItem.setIcon(util.THEME.icon("chat/rank/newplayer.png"))
-
-        self._verifySortOrder()
+            icon_str = "newplayer"
+        self.rankItem.setIcon(util.THEME.icon("chat/rank/%s.png" % icon_str))
+        self.rankItem.setToolTip(tooltip_str)
 
     def updateGame(self):
         # Status icon handling
         game = self.user_game
-        player = self.user_player
         if game is not None and not game.closed():
-            url = game.url(player.id)
+            formatting = "<br/>title: {}<br/>mod: {}<br/>map: {}<br/>players: {} / {}<br/>id: {}"
+            game_str = formatting.format(game.title, game.featured_mod, maps.getDisplayName(game.mapname),
+                                         str(game.num_players), str(game.max_players), str(game.uid))
             if game.state == GameState.OPEN:
-                self.statusItem.setIcon(util.THEME.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
+                if game.password_protected:
+                    game_str = " (private) Game Lobby:</b>" + game_str
```

Comment:
You either missed a \<b\>, or it should be a \</br\>.

Reply:
nope, was ok .. but admitted complicated